_component_design/actions-menu.md

```diff
@@ -1,17 +1,100 @@
 ---
 title: Actions Menu
-parent: Editing
-secondary: Inputs and Controls
+parent: Inputs and Controls
+secondary: Editing
 layout: component
 category: Components
 usage: >
   The actions menu is used anytime a group of actions can be performed on a specific object on the current page or the current page. By grouping these actions within the actions menu, it allows users to have a single location to trigger multiple types of actions.
```

Comment:
current page is listed twice in this sentence

Reply:
updated.